docker/Dockerfile

```diff
@@ -178,10 +178,31 @@ COPY ["*.md", "LICENSE", "./"]
 # Use morpheus by default
 CMD [ "morpheus" ]
 
+# ============ Stage: development_deps ===========
+# Configure and build cpython with debugging symbols
+FROM conda_env_dev as development_deps
+
+# Capture python version
+ARG PYTHON_VER
+COPY ci/conda/recipes/python-dbg/ ./ci/conda/recipes/python-dbg
+
+# Flag indicating if we're going to build cpython
+ARG MORPHEUS_WITH_PYDEBUG=0
+
+RUN if [[ "${MORPHEUS_WITH_PYDEBUG}" =~ ^(TRUE|True|true|1)$ ]]; then \
+      ## Figure out which python version is in the existing environment -- we will build its debug equivalent \
+      source activate morpheus \
+      && \
+        MORPHEUS_PYTHON_VER=$(python --version | cut -d ' ' -f 2) \
+        CONDA_BLD_PATH=/opt/conda/conda-bld \
+        CONDA_ARGS="--no-test" \
+      conda mambabuild -c conda-forge -c system ./ci/conda/recipes/python-dbg \
+      ; \
+    fi
+
```

Comment:
I'm not sure this makes much sense as a docker build stage here. We could easily move this functionality to a optional post-`development` build stage that is opt in instead of opt out. This will reduce the number of layers and chances to invalidate the cache and remove the need for the large `if` block. You would need to explicitly build it with something like `DOCKER_TARGET=development_pydbg ./docker/build_container_dev.sh`

Reply:
Fixed; updated to use an optional target at the end of the build.